driver/bpf/filler_helpers.h

```diff
@@ -335,7 +340,18 @@ static __always_inline u32 bpf_compute_snaplen(struct filler_data *data,
 		dport = 0;
 	}
 
-	if (sport == PPM_PORT_MYSQL || dport == PPM_PORT_MYSQL) {
+	uint16_t min_port = data->settings->fullcapture_port_range_start;
+	uint16_t max_port = data->settings->fullcapture_port_range_end;
+
+	if (min_port > 0 &&
```

Comment:
I didn't catch it earlier and just seeing this now after looking at the new commit: the code is using `min_port` here, but `max_port` in the kmod version. It might create a difference in behavior if a user wanted to put a `[0, 65535]` range to whitelist all traffic (which, incidentally, is something I'm mildly experimenting with for HTTP/2).

Other than that, all good.

Reply:
Another good catch, thanks!